### ISSUE TITLE
Hyper-V management?

### ISSUE THREAD

**Issue by dmutsaers** [MicrosoftDocs/windowsserverdocs#2059]
It would be nice to be able to manage Hyper-V locally using the Hyper-V GUI tools. (virtmgmt.msc, vmconnect) when Server App compatibilty FOD has been installed. I assumed all .msc consoles would be supported by Server App compatibilty FOD. I was wrong, however.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5da35d75-87fe-4432-9a86-d07f1b4f41b5
* Version Independent ID: 6e3f4dfd-d101-c26b-94a4-fe44093dba8e
* Content: [Server Core App Compatibility Feature on Demand (FOD)](https://docs.microsoft.com/en-us/windows-server/get-started-19/install-fod-19)
* Content Source: [WindowsServerDocs/get-started-19/install-fod-19.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/get-started-19/install-fod-19.md)
* Product: **windows-server-threshold**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by chunkyen** [comment]
Yes, you can have virtmgmt.msc now with Windows server core 1903

**Comment by Budrumi** [comment]
Why why? Can we get this for 1809 too? We're running brand new Hyper-V Server 2019, which is 1809 and Hyper-V manager cannot be used there even with FOD
